etcdserver/etcdserverpb/rpc.proto

```diff
@@ -310,6 +313,17 @@ message HashResponse {
   uint32 hash = 2;
 }
 
+message SnapshotRequest {
+}
+
+message SnapshotResponse {
+  // header has the current store information. The first header in the snapshot
+  // stream indicates the point in time of the snapshot.
+  ResponseHeader header = 1;
+  // blob has the next chunk of the snapshot in the snapshot stream.
+  bytes blob = 2;
```

Comment:
how about number of remaining bytes? I don't think offset makes sense unless it's supporting out of order messages.